src/layouts/EditContactUs.jsx

```diff
@@ -164,8 +164,17 @@ const EditContactUs = ({ match }) => {
     id: null,
     type: "",
   })
-  const [showDeletedText, setShowDeletedText] = useState(true)
   const errorToast = useErrorToast()
+  const {
+    isOpen: isRemovedOpen,
+    onOpen: onRemovedOpen,
+    onClose: onRemovedClose,
```

Comment:
i think this name might invite confusion; wdyt about something like `isRemovedContentWarningOpen` - this is more verbose but the intention is clear.

Reply:
https://github.com/isomerpages/isomercms-frontend/pull/941/commits/0721a811504d1b7b38d4d6eb61364e11db356000